pkg/network/node/vnids.go

```diff
@@ -136,6 +136,18 @@ func (vmap *nodeVNIDMap) getVNID(name string) (uint32, error) {
 	return 0, fmt.Errorf("failed to find netid for namespace: %s in vnid map", name)
 }
 
+func (vmap *nodeVNIDMap) findDuplicateNetID(namespace string, netID uint32) bool {
+	names := vmap.GetNamespaces(netID)
+	for _, name := range names {
+		if name != namespace {
+			klog.Warningf("Netid %d for namespace %s already exists under different namespace %s",
+				netID, namespace, name)
```

Comment:
sorry, the warning should be in the caller, not here. Helper functions should (ideally) not log messages

Reply:
the reason I had it here because I had access to all useful info netid, incorrect ns and the expected ns, if I moved to the caller I won't be able to have this guranularity ?